.github/workflows/tests.yml

```diff
@@ -0,0 +1,77 @@
+name: Tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  python36:
+    name: Python3.6 tests
+    runs-on: ubuntu-20.04
+
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - postgres: 10
+            python: 3.6
+            toxenv: py36
+            toxdir: psycopg3_c
+            implementation: c
+            # skip tests failing on importing psycopg3_c.pq on subprocess
+            # they only fail on Travis, work ok locally under tox too.
+            pytest_addopts: "-m 'not subprocess'"
+          - postgres: 11
+            python: 3.6
+            toxenv: py36
+            toxdir: psycopg3
```

Comment:
Toxdir is only needed because otherwise it builds a package called UNKNOWN-0.0.0. Then I have to make it find the tests using `../tests` as location. It sucks. I would be happier to just get rid of it.